docs/pipeline/actions/probabilisticsampler.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Probabilistic Sampler"
+sidebarTitle: "Probabilistic Sampler"
+---
+
+The "Probabilistic Sampler" Odigos Action supports probabilistic sampling based on a configured sampling percentage applied to the TraceID.
+
+### Use Cases
+
+#### Cost Reduction
+
+- Some vendors charge based on the amount of data ingested. For self hosted destinations, the cost is correlated to the use of cloud resources which grows with the based on the amount of data you process and store. By Reducing traces volumes, you can reduce the amount of data ingested and reduce costs.
+
+#### Usability
+
+- Performance Trend Analysis - In certain scenarios, users may seek to collect sufficient data to understand performance trends without overwhelming the system with excessive telemetry data. For example, track latency trend.
+- Telemetry Volume Management - When experiencing a sudden influx of telemetry data, probabilistic sampling can temporarily reduce data volume sent to your data destination.
```

Comment:
So do we want to have this under "Usability" after all?

Reply:
Changed, thanks